src/content/guidelines/motion/basics.mdx

```diff
@@ -1,17 +1,16 @@
 ---
-label: Motion can bring your work to life, guide users through complex experiences, and help move forward—from here to there, now to next, start to finish—and make progress.
 title: Motion
 tabs: ['Basics', 'Choreography', 'Resources']
 ---
 
 import { Tab } from 'carbon-components-react';
 
 
-### Motion is a powerful tool in designing and building communication and user experience. It can bring your work to life, guide users through complex experiences, and help move forward—from here to there, now to next, start to finish—and make progress.
+### Motion can bring the screen to life, guide users through complex experiences, and help establish the kinetic feel of the brand. 
```

Comment:
I prefer to keep the "move forward—from here to there, now to next, start to finish—and make progress." This part speaks to the philosophy of guiding users.